sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -1037,9 +1036,6 @@ def delete_blob(self, repository: str, tag_or_digest: str, **kwargs) -> None:
             from azure.identity import DefaultAzureCredential
             endpoint = os.environ["CONTAINERREGISTRY_ENDPOINT"]
             client = ContainerRegistryClient(endpoint, DefaultAzureCredential(), audience="my_audience")
-            client.delete_blob("my_repository", "my_tag_or_digest")
+            client.delete_blob("my_repository", "my_digest")
         """
-        if _is_tag(tag_or_digest):
-            tag_or_digest = self._get_digest_from_tag(repository, tag_or_digest)
-
-        self._client.container_registry_blob.delete_blob(repository, tag_or_digest, **kwargs)
+        self._client.container_registry_blob.delete_blob(repository, digest, **kwargs)
```

Comment:
This is now consistent with what other languages are doing; however, is it a breaking change to your library?

Reply:
No, this operation hasn't GAed. (The GAed one is `delete_manifest`)

Reply:
Well, it's actually a breaking change with our last preview. So it should be acceptable.

Reply:
Got it - thanks for the clarification!